source/js/components/petition/petition-thank-you.jsx

```diff
@@ -0,0 +1,180 @@
+import { Component } from "react";
```

Comment:
This is a simplified version of `petition.jsx` (which handles rendering for the current petition form, donation modal, and share buttons). For FA form, we'll only need the component to render donation modal and share buttons. I created this file by extracting relevant code here. After we fully launch the FA work, we can simply delete petition.jsx.